kubernetes-AKS/mongodb.yaml

```
@@ -0,0 +1,93 @@
+---
+# External Service for MongoDB
+apiVersion: v1
+kind: Service
+metadata:
+  name: mongodb-external-service
+  namespace: pps-namespace
+spec:
+  selector:
+    app: mongodb
+  ports:
+  - port: 27017
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)